<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Outils</class>
 <widget class="QDialog" name="Outils">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Outils</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="listeBoutonsOutils">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>560</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QListWidget" name="listeOutils">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>191</width>
     <height>318</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Gestionnaire éléments orbitaux</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mise à jour TLE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gestionnaire TLE</string>
    </property>
   </item>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget_outils">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>12</y>
     <width>751</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="gestionnaireElementsOrbitaux">
    <widget class="QWidget" name="gridLayoutWidget_11">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>290</y>
       <width>309</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_gestionnaireTLE3">
      <property name="horizontalSpacing">
       <number>3</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="ageMaxElementsOrbitaux">
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string>Age maximal des éléments orbitaux (en jours) :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="nbJoursAgeMaxElementsOrbitaux">
        <property name="font">
         <font/>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="value">
         <number>15</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget_satellites">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>-1</y>
       <width>231</width>
       <height>301</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="groupe">
      <widget class="QLabel" name="lbl_listeFichiers">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>221</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Liste des fichiers d'éléments orbitaux :</string>
       </property>
      </widget>
      <widget class="QPushButton" name="valider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>91</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
      <widget class="QPushButton" name="annuler">
       <property name="geometry">
        <rect>
         <x>114</x>
         <y>260</y>
         <width>91</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="listeFichiers">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>195</width>
         <height>161</height>
        </rect>
       </property>
       <property name="font">
        <font/>
       </property>
      </widget>
      <widget class="QWidget" name="formLayoutWidget_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>195</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_gestionnaireTLE1">
        <property name="horizontalSpacing">
         <number>3</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_domaine">
          <property name="font">
           <font/>
          </property>
          <property name="text">
           <string>Domaine :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="domaine">
          <property name="font">
           <font/>
          </property>
          <property name="inputMask">
           <string notr="true"/>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="formLayoutWidget_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>45</y>
         <width>195</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_gestionnaireTLE2">
        <property name="horizontalSpacing">
         <number>3</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_nomGroupe">
          <property name="font">
           <font/>
          </property>
          <property name="text">
           <string>Nom du groupe :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="nomGroupe">
          <property name="font">
           <font/>
          </property>
          <property name="inputMask">
           <string notr="true"/>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="satellites">
      <widget class="QListWidget" name="listeSatellites">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>18</y>
         <width>221</width>
         <height>237</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_listeSatellites">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Liste des satellites :</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QListWidget" name="listeFichiersElem">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>18</y>
       <width>221</width>
       <height>237</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_fichiersElem">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>0</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="text">
      <string>Fichiers d'éléments orbitaux :</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_groupeElem">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="text">
      <string>Groupe d'éléments orbitaux :</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listeGroupeElem">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>51</y>
       <width>221</width>
       <height>204</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="toolTip">
      <string>Cocher un groupe pour activer sa mise à jour</string>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_13">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>221</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_gestionnaireTLE4">
      <property name="horizontalSpacing">
       <number>3</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_serveur">
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string>Domaine :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="serveur">
        <property name="currentText">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="majGroupe">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>262</y>
       <width>221</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Mettre à jour le groupe sélectionné</string>
     </property>
    </widget>
    <widget class="QToolButton" name="creationGroupe">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="ajoutFichiersElem">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>-1</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QFrame" name="frameBarreProgressionElem">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>262</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QProgressBar" name="barreProgressionElem">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="format">
       <string notr="true">%p%</string>
      </property>
     </widget>
     <widget class="QLabel" name="fichierTelechargementElem">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font/>
      </property>
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="vitesseTelechargementElem">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="miseAJourTLE">
    <widget class="QTabWidget" name="majTLE">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>741</width>
       <height>301</height>
      </rect>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="majTLEauto">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Mise à jour TLE auto</string>
      </attribute>
      <widget class="QPlainTextEdit" name="compteRenduMajAuto">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>44</y>
         <width>715</width>
         <height>221</height>
        </rect>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QFrame" name="frameBarreProgressionTLE">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>10</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <widget class="QProgressBar" name="barreProgressionTLE">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>161</width>
          <height>23</height>
         </rect>
        </property>
        <property name="font">
         <font/>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
        <property name="format">
         <string notr="true">%p%</string>
        </property>
       </widget>
       <widget class="QLabel" name="fichierTelechargementTLE">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="vitesseTelechargementTLE">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>0</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="majMaintenant">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Mettre à jour maintenant</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="affichageMsgMAJ">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>10</y>
         <width>241</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="currentText">
        <string notr="true">Affichage des messages informatifs</string>
       </property>
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Affichage des messages informatifs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Accepter ajout/suppression de TLE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Refuser ajout/suppression de TLE</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="majTLEmanuelle">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Mise à jour TLE manuelle</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>681</width>
         <height>54</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_majTle">
        <property name="horizontalSpacing">
         <number>3</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="1" column="2">
         <widget class="QPushButton" name="parcourirMaj2">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font/>
          </property>
          <property name="text">
           <string>Parcourir...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_fichierAMettreAJour">
          <property name="font">
           <font/>
          </property>
          <property name="text">
           <string>Fichier à mettre à jour :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="fichierALire">
          <property name="font">
           <font/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="parcourirMaj1">
          <property name="font">
           <font/>
          </property>
          <property name="text">
           <string>Parcourir...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="fichierAMettreAJour">
          <property name="font">
           <font/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lbl_fichierALire">
          <property name="font">
           <font/>
          </property>
          <property name="text">
           <string>Fichier TLE à lire :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="mettreAJourTLE">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font/>
          </property>
          <property name="text">
           <string>Mettre à jour</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPlainTextEdit" name="compteRenduMajManuel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>715</width>
         <height>195</height>
        </rect>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="gestionnaireTLE">
    <widget class="QListWidget" name="listeTLE">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>18</y>
       <width>221</width>
       <height>281</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="supprimerTLE">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_listeTLE">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Liste de fichiers TLE :</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frameInformations">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>792</width>
     <height>226</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>listeOutils</zorder>
  <zorder>stackedWidget_outils</zorder>
  <zorder>listeBoutonsOutils</zorder>
  <zorder>frameInformations</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>listeBoutonsOutils</sender>
   <signal>accepted()</signal>
   <receiver>Outils</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listeBoutonsOutils</sender>
   <signal>rejected()</signal>
   <receiver>Outils</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
